### PR TITLE
updated vignette to reflect Moonlight2 in greater detail

### LINKED ISSUE #26: Update vignette to reflect Moonlight2 in greater extent
Update the vignette to reflect Moonlight2 in greater extent by:
- editing author list
- updating title
- updating abstract
- updating introduction
- update figure of Moonlight pipeline
- update citations to replace TCGAbiolinks citations with Moonlight citations
- keep only filename of Cscape files in DMA function
- add a case study 4 to reflect DMA

### PR DESCRIPTION
**Pull request by mona-n**
fixes #26

### CODE REVIEW

**Review by mtiberti — CHANGES_REQUESTED** [review]
one additional point

- this sentence is unfinished: For methylation data, DPA is running a differentially methylated regions analysis (DMR) to identify differentially methylated CpG sites using the TCGAanalyze_DMR the TCGAanalyze_DMR function from .

File: vignettes/Moonlight2R.Rmd [hunk 1]
```diff
@@ -378,47 +397,69 @@ devtools::load_all(".")
...
 
 # Moonlight's proposed workflow
 
-The proposed pipeline consists of following eight steps:
+The proposed pipeline consists of following nine steps:
 
 1. **getDataTCGA** \& **getDataGEO** for Data collection: expression levels of genes in all samples obtained with IlluminaHiSeq RNASeqV2 in 18 normal tissues (NT) and 18 cancer tissues (CT) according to TCGA criteria, and GEO data set matched to one of the 18 given TCGA cancer types as described in following Table TCGA / GEO.
 2. **DPA** Differential Phenotype Analysis (DEA) to identify genes or probes that are different significantly with two phenotypes such as normal and tumor, or normal and stageI, normal and molecular subtype.
-3. **FEA** Functional Enrichment Analysis (EA), using Fisher's test, to identify gene sets (with biological functions linked to cancer<sup>1</sup>) significantly enriched by RG.
+3. **FEA** Functional Enrichment Analysis (EA), using Fisher's test, to identify gene sets (with biological functions linked to cancer) significantly enriched by RG.
```
[the remark this answers is not in the document]
Comment (mona-n) [reply vignettes/Moonlight2R.Rmd]: updated:
removed FEA abbreviation for consistency in text as abbreviations are not mentioned in the other places
defined RG acronym

File: vignettes/Moonlight2R.Rmd [hunk 2]
```diff
@@ -378,47 +397,69 @@ devtools::load_all(".")
...
 grid.raster(img)
 ```
 
 # Moonlight's proposed workflow
 
-The proposed pipeline consists of following eight steps:
+The proposed pipeline consists of following nine steps:
 
 1. **getDataTCGA** \& **getDataGEO** for Data collection: expression levels of genes in all samples obtained with IlluminaHiSeq RNASeqV2 in 18 normal tissues (NT) and 18 cancer tissues (CT) according to TCGA criteria, and GEO data set matched to one of the 18 given TCGA cancer types as described in following Table TCGA / GEO.
 2. **DPA** Differential Phenotype Analysis (DEA) to identify genes or probes that are different significantly with two phenotypes such as normal and tumor, or normal and stageI, normal and molecular subtype.
```
[the remark this answers is not in the document]
Comment (mona-n) [reply vignettes/Moonlight2R.Rmd]: updated:
removed DEA abbreviation for consistency with the rest of the text
updated "are different significantly" to "are significantly different"

File: vignettes/Moonlight2R.Rmd [hunk 3]
```diff
@@ -378,47 +397,69 @@ devtools::load_all(".")
...
...ped the Moonlight framework that allows for prediction of cancer driver genes (Colaprico and Olsen et al., 2020). Here, gene expression data are integrated together with biological processes and gene regulatory networks to score the importance of well-known biological processes with respect to the studied cancer. These scores are used to predict  oncogenic mediators: putative TSGs and putative OCGs. As gene expression data alone is not enough to explain the deregulation of the genes, a second layer of evidence is needed. For this reason, we automated the integration of a secondary mutational layer which predicts driver mutations and passenger mutations in the oncogenic mediators. These new functionalities are released in the updated version of Moonlight to produce Moonlight2R. The prediction of the driver mutations are carried out using the CScape-somatic driver mutation prediction tool. Moreover, the new functionalities estimate the potential effect of a mutation on the transcriptional, translational, or protein structure/function level. Those oncogenic mediators with at least one driver mutation are retained as the final set of driver genes (Saksager and Nourbakhsh et al., 2022).
```
[the remark this answers is not in the document]
Comment (mona-n) [reply vignettes/Moonlight2R.Rmd]: updated both

File: vignettes/Moonlight2R.Rmd [hunk 4]
```diff
@@ -378,47 +397,69 @@ devtools::load_all(".")
...
+7. **DMA** Driver Mutation Analysis identifies driver mutations in the putative TSGs and OCGs through the driver mutation prediction tool, CScape-somatic.
+8. We applied the above procedure to multiple cancer types to obtain cancer-specific lists of TSGs and OCGs. We compared the lists for each cancer: if a sDEG was TSG in a cancer and OCG in another we defined it as dual-role TSG-OCG. Otherwise if we found a sDEG defined as OCG or TSG only in one tissue we defined it tissue specific biomarker.
+9. We use the COSMIC database to define a list of gold standard TSG and OCGs to assess the accuracy of the proposed method.
 
-<sup>1</sup> For the devel version of Moonlight2R we use a short extract of ten biological functions from QIAGEN'S Ingenuity Pathway Analysis (IPA). We are still working to integrate the\Biocpkg{ReactomePA} package.
 
 # Installation
 
-To install use the code below.
+To install Moonlight2R use the code below.
```
[the remark this answers is not in the document]
Comment (mona-n) [reply vignettes/Moonlight2R.Rmd:477]: updated

File: vignettes/Moonlight2R.Rmd [hunk 5]
```diff
@@ -427,11 +468,15 @@ library(tidyverse)
 
 ## Citation
```
[the remark this answers is not in the document]
Comment (mona-n) [reply vignettes/Moonlight2R.Rmd]: updated:
citation section moved to the end before bibliography

I'm not sure how to get a full citation automatically and which part this refers to?

File: vignettes/Moonlight2R.Rmd [hunk 6]
```diff
@@ -378,47 +397,69 @@ devtools::load_all(".")
...
 
-<sup>1</sup> For the devel version of Moonlight2R we use a short extract of ten biological functions from QIAGEN'S Ingenuity Pathway Analysis (IPA). We are still working to integrate the\Biocpkg{ReactomePA} package.
 
 # Installation
 
-To install use the code below.
+To install Moonlight2R use the code below.
 
 ```{r, eval = FALSE}
 devtools::install_github(repo = "ELELAB/Moonlight2R")
```
[the remark this answers is not in the document]
Comment (mona-n) [reply vignettes/Moonlight2R.Rmd:488]: updated

File: vignettes/Moonlight2R.Rmd [hunk 7]
```diff
@@ -378,47 +397,69 @@ devtools::load_all(".")
...
 ```
 
+To install Moonlight2R with its accompanying vignette use the code below.  
+
+First, install the BiocStyle Bioconductor package.
+
+```{r, eval = FALSE}
+if (!require("BiocManager", quietly = TRUE))
```
[the remark this answers is not in the document]
Comment (mona-n) [reply vignettes/Moonlight2R.Rmd:496]: updated all

File: vignettes/Moonlight2R.Rmd [hunk 8]
```diff
@@ -638,8 +683,8 @@ dataPRA <- PRA(dataURA = dataURA,
 ```
 
 ## `DMA`: Driver Mutation Analysis
-The user can identify driver mutations with `DMA` in the oncogenic mediators established by `PRA`.  The passenger or driver status is estimated with Cscape-somatic.  
-This function will further generate three files: DEG_Mutations_Annotation.rda, Oncogenic_mediators_mutation_summary.rda and cscape_somatic_output.rda. These will be placed in the specified results-folder. 
+The user can identify driver mutations with `DMA` in the oncogenic mediators established by `PRA`.  The passenger or driver status is estimated with CScape-somatic.  
+This function will further generate three files: DEG_Mutations_Annotations.rda, Oncogenic_mediators_mutation_summary.rda and cscape_somatic_output.rda. These will be placed in the specified results-folder.
```
[the remark this answers is not in the document]
Comment (mona-n) [reply vignettes/Moonlight2R.Rmd]: updated

**Pull request merged**